sdk-core/src/main/java/com/reach5/identity/sdk/core/models/User.kt

```diff
@@ -1,13 +1,12 @@
 package com.reach5.identity.sdk.core.models
 
-import android.os.Parcel
 import android.os.Parcelable
 import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
 
+@Parcelize
 data class User(
-    /**
-     * Correspond au champ "sub" dans le JSON
-     */
+    // Correspond au champ "sub" dans le JSON
```

Comment:
Should we translate the comments into english for the SDK ?

Reply:
That's a good point. I'll do it on my next PR.